### PR TITLE
FFI wrapper for PIAL

### PR DESCRIPTION
**Pull request by ParnianAlimi**
Addresses CRYP-198 and CRYP-200.

### CODE REVIEW

File: private-identity-audit/ffi/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+[lib]
+name = "private_identity_audit_ffi"
+crate-type = ["cdylib"]
+
+[features]
+default = ["cbindgen", "std"]
+no_std = []
+std = [
+    # "serde_all",
```
Comment (aafshar-poly) [inline private-identity-audit/ffi/Cargo.toml:17]: Remove unused dependencies.

File: private-identity-audit/ffi/examples/c_example.c [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+#include <inttypes.h>
+#include <string.h>
+#include "c_example.h"
+
+/**
+ * Creates a CDD ID from a CDD claim.
+ * (copy/pasted here from confidential-identity/ffi/examples/c_example.h
+ *  for convenience.)
+ *
+ * SAFETY: Caller is responsible to make sure `cdd_claim` pointer is a valid
```
Comment (aafshar-poly) [inline private-identity-audit/ffi/examples/c_example.c:12]: Please run clippy on these files. Clippy requires a specific format for the "safety" messages.

**Review by aafshar-poly — APPROVED** [review]
Look good to me. Feel free to merge after running clippy.

File: confidential-identity/cli/README.md [hunk 3]
```diff
@@ -79,31 +86,30 @@ polymath-scp create-cdd-id -h
 
 ### Investors' Usage
 To run Claim prover CLI:
+>>>>>>> ec8846e01282024f9a2ce65721826a9d779261b3
```
Comment (aafshar-poly) [inline confidential-identity/cli/README.md:89]: leftover from merge.

File: private-identity-audit/ffi/examples/c_example.c (hunk 2 above)
Comment (ParnianAlimi) [reply private-identity-audit/ffi/examples/c_example.c:12]: What did you run to see that formatting issue? just `cargo clippy`? It gives me irrelevant errors.

Comment (ParnianAlimi) [reply private-identity-audit/ffi/examples/c_example.c:12]: nvm

**Review by aafshar-poly — APPROVED** [review]

**Pull request merged**